com.playeveryware.eos/CHANGELOG.md

```diff
@@ -4,6 +4,282 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+  - chore(lib,android) : remove unused android libs
+  - chore(sdk,source) : updated managed source for 1.16.3 hotfix
+  - chore(sdk,lib) : updated binaries for 1.16.3 hotfix
+  - chore(eac,tool) : Allow anticheat integrity tool file path to recognize executables with no extension
+  - chore: Update eos_package_description.json with new location of core code
+  - chore: Update eos_package_description.json with new location of EOS SDK.
+  - chore: Update eos_package_description.json with new location of docs
+  - chore: Update eos_package_description.json with new location of PackageTemplate files
+
+- Sample Rework
+  - rework(auth): Better variable names, namespace using moved, null propagation, 0-safe division
+  - rework(auth): Fix signature of UILoginMenu call
+  - rework(auth): Retry attempt count, remove "WithDiscord", camel case variable name
+  - rework(session): Fix to Test logic
+  - rework(sessions): Renamed Tests
+  - rework(sessions): Removing unused test function
+  - rework(session): Define Session in Out Parameter of Function Call
+  - rework(sessions): Demoting Error to Warning for no local Session found on informing owner
+  - rework(sessions): SessionsManager Callbacks
+  - rework(sessions): Reset and Init
+  - rework(sessions): OnShutDown removed, no longer mentioned in code comments
+  - rework(sessions): Decapitalize "R" in "Unregister"
+  - rework(sessions): `dirtyFlag` -> `IsDirty`
+  - rework(sessions): `dirtyFlag` -> `IsDirty`
+  - rework(sessions): Decapitalize "R" in "Unregister"
+  - rework(sessions): Rearranging Manager Code into Regions
+
+- Documents and Comments
+  - chore (Sessions): Standardizing comments inside EOSSessionsManager
+  - chore (EOSSessionsManager): Change constants to SCREAMING_SNAKE_CASE per standards
+  - chore(comment) : update comment to better describe config write use cases
+  - chore (Sessions): Standardizing comments inside EOSSessionsManager
+  - chore (EOSSessionsManager): Change constants to SCREAMING_SNAKE_CASE per standards
+  - chore (documentation, Lobby): Return values for messages documented, some wording changes to params
+  - chore (documentation, Lobby): Adding clarifying documentation mentioning subscription
+  - docs(sessions): Some touchup to Session capitalization
+  - docs (sessions): Responding to PR Feedback
+  - docs (Sessions): Slight commenting tweak
+  - docs (Sessions): Some additional commenting tweaks
+  - docs (Sessions): Continued effort in commenting everything
+  - docs (Sessions): More comments
+  - docs (sessions): Writing comments for functions and regions, part one
+  - docs(sessions): Some touchup to Session capitalization
+  - docs: update links to reference new doc location
+  - docs (Sessions): Commenting Functions
+  - docs (sessions): Responding to PR Feedback
+  - docs (Sessions): Slight commenting tweak
+  - docs (Sessions): Some additional commenting tweaks
+  - docs (Sessions): Continued effort in commenting everything
```

Comment:
These lines about commenting could be condensed into 'add additional commenting'